sockeye/arguments.py

```diff
@@ -1109,3 +1121,51 @@ def add_init_embedding_args(params):
                         help='File to write initialized parameters to.')
     params.add_argument('--encoding', '-c', type=str, default=C.VOCAB_ENCODING,
                         help='Open input vocabularies with specified encoding. Default: %(default)s.')
+
+def add_scoring_args(params):
+    scoring_params = params.add_argument_group("Scoring parameters")
+
+    scoring_params.add_argument('--source', '-s',
+                               required=True,
+                               type=regular_file(),
+                               default=None,
+                               help='Source file to be scored. One sentence per line.')
+    scoring_params.add_argument('--target', '-t',
+                               required=True,
+                               type=regular_file(),
+                               default=None,
+                               help='Target file to be scored. One sentence per line.')
+    scoring_params.add_argument('--models', '-m',
+                               required=True,
+                               nargs='+',
+                               help='Model folder(s). Use multiple for ensemble scoring.')
+    scoring_params.add_argument('--checkpoints', '-c',
+                               default=None,
+                               type=int,
+                               nargs='+',
+                               help='If not given, chooses best checkpoints for model(s). '
+                                    'If specified, must have the same length as --models and be integer')
+    scoring_params.add_argument('--output-type',
```

Comment:
I guess one use case would be to plot the alignment of a given translation, no? (not sure how important it is and would probably be fine either way)

Reply:
I added this argument because we could probably extend the list of handlers in the future. Yes, plotting the alignment would be useful, either producing plots directly or output a format that can be turned into plots (https://github.com/M4t1ss/SoftAlignments#how-to-get-alignment-files-from-nmt-systems). That said, only RNN models have attention scores right now and I can also drop the argument for now.